src/main/java/htsjdk/variant/variantcontext/VariantContextBuilder.java

```diff
@@ -342,23 +356,52 @@ private void makeAttributesModifiable() {
      * @return this builder
      */
     public VariantContextBuilder filters(final Set<String> filters) {
-        this.filters = filters;
+        makeFiltersModifiable();
```

Comment:
Maybe a better approach would be to pass `filters` to `makeFiltersModifiable()`? It seems like you've got similar logic split across different methods here.

Reply:
I'm not sure I understand what you mean. I see that the pattern of `makeFiltersModifiable` is identical to that of `makeAttributesModifiable`, but making one method out of that seems a bit contrived....

Reply:
oh. nm, I did this :-)